cloudformation-templates/pattern-1/testgrid.yaml

```diff
@@ -0,0 +1,36 @@
+# TestGrid configuration file.
```

Comment:
Why do we have a ```testgrid.yaml``` in the cf-ei repo? I don't think a customer trying to setup EI on AWS would need this yaml file.

Reply:
This is used as config file to run the deployment on TestGrid. This will not affect the rest of the files in the repo. This was agreed upon to have in the repository after discussions.

Reply:
@msmshariq, Yes, as you said, customers do not need to use this. 

This file is quite similar to configuration files used by other CI servers in the industry. For example, Jenkins CI uses [Jenkinsfile](https://github.com/search?l=groovy&o=desc&q=pipeline+mvnHome&s=&type=Code) and Travis CI uses [.travis.yaml](https://github.com/search?o=desc&q=mvn+oraclejdk8&s=indexed&type=Code).

The sole function of these files is describing a repo. It describes the environment it needs, and how it should be built. 

For WSO2 infrastructure, deployment, and scenario test repos, we need a same configuration file that describe those repos. For Terraform repos, this file would mention that this is a Terraform repo, and it can run on these these cloud providers (say AWS and GCP). In addition, it'll also mention what test scripts we can run to provision a deployment in those cloud providers.

I had a discussion with @chamithkumarage. We agreed that having this file makes sense, but we need to improve the naming since customers has no use for this file. Therefore, as discussed with you offline, we finalized to make this a hidden file by prepending a dot '.' . 

With that, the new name is "**.testgrid.yaml**".

Reply:
Yes, makes sense. +1 for the new naming convention.